core-bundle/src/Resources/contao/library/Contao/Search.php

```diff
@@ -173,7 +173,7 @@ public static function indexPage($arrData)
 		$strBody = strip_tags($strBody);
 
 		// Put everything together
-		$arrSet['text'] = $arrData['title'] . ' ' . $arrData['description'] . ' ' . $strBody . ' ' . $arrData['keywords'];
+		$arrSet['text'] = $strBody . "\n" . $arrData['title'] . ' ' . $arrData['description'] . ' ' . $arrData['keywords'];
```

Comment:
```suggestion
		$arrSet['text'] = $strBody . ' ' . $arrData['description'] . "\n" . $arrData['title'] . "\n" . $arrData['keywords'];
```
As google also sometimes shows the description in search results, I think we can do that too. The title will always be shown, so it does not need to be in the context I agree.

Reply:
did you intentionally move title and keywords to separate lines?

Reply:
Yes so that the structure of the column is `[text]\n[title]\n[keywords]` as it might be useful to be able to distinguish between title and keywords IMO.